SharpPcap/LibPcap/LibPcapSafeNativeMethods.Interop.cs

```diff
@@ -41,7 +41,7 @@ internal static partial class LibPcapSafeNativeMethods
         //       This file is called $assembly_name.dll.config and is placed in the
         //       same directory as the assembly
         //       See http://www.mono-project.com/Interop_with_Native_Libraries#Library_Names
-        private const string PCAP_DLL = "wpcap";
+        private const string PCAP_DLL = "pcap";
```

Comment:
wpcap is required as default name for compatibility with .NET Framework

Reply:
Done.